lisa/sut_orchestrator/azure/common.py

```diff
@@ -2734,6 +2734,9 @@ def get_managed_identity_object_id(
 def get_identity_id(
     platform: "AzurePlatform", application_id: Optional[str] = None
 ) -> Any:
+    if not application_id:
+        application_id = os.environ.get("AZURE_CLIENT_ID")
```

Comment:
Make sure it won't fail, if the variable doesn't exist in the environment, like `os.environ.get("AZURE_CLIENT_ID", "")`

Reply:
updated